src/pages/MediaManip/DragDrop/DragDrop.module.scss

```diff
@@ -0,0 +1,56 @@
+.container {
+  width: 70%;
+  min-height: 8rem;
+  height: max-content;
+  margin: 2rem auto;
+  border: 2px solid #cccccc;
+  display: flex;
+  justify-content: flex-start;
+  border-radius: 5px;
+
+  &_image {
```

Comment:
Why is '&' used here?

Reply:
@anushbhatia it's scss and it's a way of selecting class name.

If parent div have a class name of container and a child element have a classname of container_image then after defining styles for container we can select child by &_image. 
In this context &_ means container.

.container {
  
   &_image{
        }
}